data/json/itemgroups/locations.json

```diff
@@ -78,4 +78,3 @@
     ]
   }
 ]
- 
```

Comment:
On my copy they are still there. There was an additional extra new line at the end of the file.